validator/client/validator_attest.go

```diff
@@ -172,7 +146,10 @@ func (v *validator) SubmitAttestation(ctx context.Context, slot uint64, pubKey [
 		}
 		return
 	}
-
+	indexedAtt.Signature = sig
+	if err := v.postSignatureUpdate(ctx, indexedAtt, pubKey); err != nil {
+		return
```

Comment:
Can we log something out or handle the error? Maybe we should log here instead of in the other function?

Reply:
did it here